db/record.c

```diff
@@ -88,6 +89,16 @@ void free_cached_idx(uint8_t * *cached_idx);
     if (gbl_verbose_toblock_backouts)                                          \
         logmsg(LOGMSG_USER, "err line %d rc %d retrc %d\n", __LINE__, rc, retrc);           \
     goto err;
+#define VERIFY_TABLE_VERSION                                                   \
+    if (iq->usedb->tableversion != iq->usedbtablevers) {                       \
+        if (iq->debug)                                                         \
+            reqprintf(iq, "Stale buffer: usedb version %d vs curr ver %d\n",   \
+                      iq->usedbtablevers, iq->usedb->tableversion);            \
+        poll(NULL, 0, BDB_ATTR_GET(thedb->bdb_attr, SC_DELAY_VERIFY_ERROR));   \
```

Comment:
Don't mean to be picky, but we should really use usleep to sleep, and poll to poll (always disliked using poll to sleep--it's just confusing).